src/TextareaAutosize.js

```diff
@@ -127,7 +129,7 @@ export default class TextareaAutosize extends React.Component {
   _resizeComponent() {
     let {useCacheForDOMMeasurements} = this.props;
     this.setState(calculateNodeHeight(
-      React.findDOMNode(this),
+      ReactDOM.findDOMNode(this),
```

Comment:
I think can make it work completely without `react-dom` package if we set a `ref` on component's DOM node which in React 0.14 will return a DOM node. So we don't need `ReactDOM.findDOMNode` at all.